MSAL/src/MSALPublicClientApplication.m

```diff
@@ -1501,4 +1522,14 @@ - (MSIDRequestParameters *)defaultRequestParametersWithError:(NSError **)request
     return requestParams;
 }
 
+- (MSIDDevicePopManager *)popManager
```

Comment:
you're creating pop manager in any case (even for bearer). Might as well create it always in initializer.

Reply:
Addressed.